src/components/license/LicenseBox.tsx

```diff
@@ -139,6 +139,7 @@ const LicenseBox = ({ article, copyText, printUrl, oembed }: Props) => {
   const tabs = buildLicenseTabList(article, t, copyText, printUrl, oembed);
   return (
     <TabsRoot
+      css={{ paddingBlockEnd: "medium" }}
```

Comment:
Kanskje heller lage en `StyledTabsContent` så slipper vi å sende inn css på denne måten til `TabsRoot`? Siden dette er en special case for akkurat _Sitere eller gjenbruke_ @Jonas-C